app/api/server/v1/channels.js

```diff
@@ -588,8 +593,13 @@ API.v1.addRoute('channels.members', { authRequired: true }, {
 		const total = subscriptions.count();
 
 		const members = subscriptions.fetch().map((s) => s.u && s.u._id);
+		const query = {
+			name,
```

Comment:
Is expected that `name` and `username` are not regexp?

Reply:
Yes, `name` and `username` shouldn't be regexp.